.github/workflows/pr-github-checks.yml

```diff
@@ -11,7 +11,7 @@ on:
       - edited
       - synchronize
       - labeled
-      - milestoned
+      - unlabeled
```

Comment:
why not milestoned?

Reply:
According to the documentation, that type is only available for issues: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#issues
And my IDE is saying that it is an unexpected value